simbad/util/mtz_util.py

```diff
@@ -226,7 +225,7 @@ def check_anomalous(self, miller_array):
         elif miller_array.info().type_hints_from_file == 'anomalous_difference':
             return True
         # Check for anomalous miller arrays which aren't properly labeled
-        elif 'DANO' in miller_array.info().labels:
+        elif any(['DANO' in i.upper() for i in miller_array.info().labels]):
```

Comment:
I think you're doing this call twice, put it in a function and ideally add test cases

Reply:
I'm not sure that I'm doing anything twice here,but admittedly the code is a bit messy. I'll refactor it.